formatTest/unit_tests/expected_output/bucklescript.re

```diff
@@ -0,0 +1,88 @@
+bla#=10;
+
+bla#=(Some 10);
+
+bla#=(someFunc (Some 10));
+
+test##var#=(Some (-10));
+
+obj##.prop;
+
+obj##.prod := exp;
+
+preview##style##border#=(
+                          Js.string "1px black dashed"
+                        );
+
+preview##(style##border)#=args somenum;
+
+x##y##z#=(xxxx##yyyy##zzzz);
+
+let result =
+  js_method_run1 (!react)#createElement foo;
+
+add zz##yy xx##ww;
+
+/* These should print the same */
+let res = x##y + z##q; /* AST */
+
+let res = x##y + z##q; /* Minimum parens */
+
+/* These should print the same */
+let res = y + z##q##a; /* AST */
+
+let res = y + z##q##a; /* Min parens */
+
+/* Make sure it's actually parsed as left precedence
+ * and that is maintained when printed */
+let res = z##(q##a); /* AST */
+
+let res = z##(q##a); /* Min parens */
+
+/* These should print the same */
+let res = !x##y; /* AST */
+
+let res = !x##y; /* Minimum parens */
+
+/* These should print the same */
+let res = !z##q##a; /* AST */
+
+let res = !z##q##a; /* Min parens */
+
+/* These should print the same */
+let res = ?!!x##y; /* AST */
+
+let res = ?!!x##y; /* Minimum parens */
+
+/* These should print the same */
+let res = ?!!z##(q##a); /* AST */
+
+let res = ?!!z##(q##a); /* Min parens */
+
+res#=?!!z##q;
+
+res#=?!!z##(q##a);
+
+let result = myFunction (x y##z) (a b#=c);
+
+(!x)##y##(b##c);
+
+let a = {"key": 10};
+
+let b = {
+  "nested": {"objs": {"are": {"nice": "<3"}}}
+};
+
+let c = {
+  "a",
+  "b",
+  "func": fun a => a##c#=(func 10)
+};
+
+let d = {
+  "a": a2,
+  "b", /* punned */
```

Comment:
Nice! But the reason that this is justified for records, was because no one (okay hardly anyone) ever has a record with only one field. However, with objects, I think it's much more likely because objects are extensible (in terms of their rows) and you can make a function that expects an object (that has a name, and possibly more) which encourages people to make an object with just the one/two fields required by the function.